web/modules/custom/joinup_group/src/Form/ShareFormBase.php

```diff
@@ -145,12 +146,12 @@ protected function getAlreadySharedCollectionIds(): array {
    *   An array of groups.
```

Comment:
Same one line up ^ (I cannot select that line for commenting) s/RdfInterface/GroupInterface